careers/software-engineer.md

```diff
@@ -4,52 +4,56 @@ The goal of the Software Engineer path is to train engineers to master the funda
+**Technical Knowledge:** They have a robust understanding of their technical specialty, and a working knowledge of other domains. They can solve most problems in their domain. Their code is well-abstracted and performant as a rule and rarely needs a significant refactor at the PR stage. If an unexpected issue/bug does arise, they see developing a solution to that problem as their personal responsibility.
 
-**Estimation:** An Engineer III Is able to provide reliable estimates for features that take longer than a single sprint to complete. Consistently delivers their work within a small margin of error to their original estimation, allowing the team to plan effectively and rely on the timely delivery of work.
+**Productivity:** They are capable of delivering a large volume of work regardless of the complexity/uncertainty of the work. They provide reliable estimates for features that take longer than a single sprint to complete.
 
-**Communication:** An Engineer III recognizes areas in our process that are redundant, manual, slow down work negatively, or allow for untested/unverified code to be pushed to live systems and offers suggestions to improve the process. An Engineer III knows that their work technically never "ends" in the sense that they always have the opportunity to recognize other opportunities for improvement in the platform and work down technical debt. They regularly examine our codebase, use our product, look at our backlog looking for the most important areas where they can make an impact.
+**Impact:** An Engineer III takes ownership over the "how" of the feature they implement with a "buck stops here" attitude. They are able to independently plan work for multiple engineers that will take several sprints to complete. They can identify strategic opportunities and risks with a proposed plan and affect the direction of the team. 
 
-## Credits
+**Communication:** They point out novel opportunities for refactoring during code review. They proactively work to minimize or mitigate potential problems before they arise.
 
-Packback's engineering ladder was inspired by documentation published by [Rent the Runway](https://dresscode.renttherunway.com/blog/ladder) and [Kickstarter](https://kickstarter.engineering/the-kickstarter-engineering-and-data-team-ladder-96996c3b327) on their engineering ladders. Thanks to them for open-sourcing that information.
+**Collaboration:** Others on the team reach out for help in their area of expertise, and they make themselves available. They are capable of effectively running small to medium-sized projects independently, but need coaching to successfully manage large projects.
```

Comment:
`They are capable of effectively running small to medium-sized projects independently` - what does "running the project" mean in this context? I feel like other descriptions at Eng II suggest being a DRI at that level, so is this also in reference to being a DRI? 

I worry about the use of "independently" here because this is in the "Collaboration" section and there is a lot of collaboration with other team members when running any project.

Reply:
Yeah, again, leave the language open. DRI is one path, but it's not the only path.

Re: "independently" ... I guess what I'm trying to differentiate here is that they should have the skills to effectively run medium-sized projects effectively. If they're working on a large+ project, they'll probably need coaching from someone more senior to do it effectively.

Is there a better way to word that?

Reply:
I guess it just feels a little odd because in the context of a DRI, the Lead Reviewer is kind of meant to be someone more senior that could offer that coaching, right? So hopefully every DRI would have a coach that could help them, and "independently" feels like you're doing it on your own. Maybe we could reword it to something like:

`They are capable of effectively running small to medium-sized projects with little to no coaching, but need help to successfully manage large projects`?